tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
```

Comment:
"changing mindset" of who? what?

Reply:
Update, same PR as above, with this suggestion:
All activities of this TAG should aim to positively impact the community by working to help reach net-zero goals and simplifying the processes to do so.

Reply:
way better. thanks

Reply:
> would recommend adding something to the effect of "and strategizing areas of incentive for service providers to reduce their consumption and carbon footprint through cloud native tooling."

Could you explain what you mean here? I am not clear on "Strategizing areas of incentive". Are you suggesting a standard?

Reply:
definitely not standards.  often ive found some groups dont see why they should do something. laying that out for them in an easy way where they understand the value and are subsequently incentivized to apply the change is real valuable in this space, particularly where there is cost savings long term or immediate after initial investment of the change

Reply:
Ok, so maybe revise to: " "and identifying value(s) and possible incentives for service providers to reduce their consumption and carbon footprint through cloud native tooling."

Reply:
Excellent!

Reply:
Updated.  many thanks.